lib/task.js

```diff
@@ -149,6 +150,22 @@ function factory(
             });
     };
 
+    Task.prototype.getPath = function(obj, path) {
```

Comment:
Seems the lodash `get` can achieve the same functionality with `getPath`: http://devdocs.io/lodash~4/index#get